__tests__/integration.test.js

```diff
@@ -38,13 +39,53 @@ describe('GET: /api/topics', () => {
     })
 })
 
-describe('GET: /api', () => {
-    test('GET200: endpoint responds with a JSON where each object has ', () => {
+describe('GET /api', () => {
+    test('GET200: endpoint responds with a JSON that describes all endpoints', () => {
         return request(app)
         .get("/api")
         .expect(200)
         .then(({body})=>{
             expect(body).toEqual(endpoints)
         })
     })
+})
+
+describe('GET /api/articles/:article_id', () => {
+    test('GET200: endpoint responds with an article object with the correct properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body})=>{
+            const article = body
+            expect(typeof article.author).toBe('string')
```

Comment:
have a look at .toMatchObject and .objectContaining here for a better way to assert against objects, this is a bit hard to read